migration/idempotent/007-tracing-tags.sql

```diff
@@ -307,10 +310,11 @@ BEGIN
     AS
```

Comment:
As you're dropping the operator class, can you schema-qualify the `btree_tag_v_ops` operator class?